sdk/storage/azure-storage-blobs/samples/blob_sas.cpp

```diff
@@ -5,19 +5,15 @@
 
 #include <azure/storage/blobs.hpp>
 
-#include "samples_common.hpp"
-
-SAMPLE(BlobSas, BlobSas)
-void BlobSas()
+int main()
 {
   using namespace Azure::Storage::Blobs;
 
-  std::string accountName = GetAccountName();
-  std::string accountKey = GetAccountKey();
-
-  std::string containerName = "sample-container";
-  std::string blobName = "sample-blob";
-  std::string blobContent = "Hello Azure!";
+  const std::string accountName = "";
+  const std::string accountKey = "";
```

Comment:
Read this from env-vars as well.  The current `test-resources` defines  `ACCOUNT_NAME` and `ACCOUNT_KEY`.  We should consider updating this to `AZURE_STORAGE_ACCOUNT_NAME` and `AZURE_STORAGE_ACCONT_KEY`.

Reply:
I'll just read from `AZURE_STORAGE_CONNECTION_STRING` and parse account name and key by myself.